README.md

```diff
@@ -202,7 +202,7 @@ installed on your local machine, which can be the same device running OBS Studio
     To authenticate using OpenVPN, connect with your OpenVPN client using `client.conf` or
     `client.ovpn`, then publish your OBS Studio stream to:
 
-        rtmp://10.10.10.1:1935/live
+        rtmp://10.10.10.1:1935/live/stream1
```

Comment:
Can we choose something other than `stream1`? Is this meant for multi-stream support?

Reply:
You can choose anything you want.  

Support for multiple streams is trivial at this point.  Right now process stream is locked on to process "stream1.m3u8"  however it can easily either be spawned as a parallel process for each new stream, or simply written to look for streamN.m3u8 and place the fines in the respective liveN.m3u8 file.

This was a possibility for multiple rooms at Our Networks, however as it seems its not required its not been completed.

Issue in https://github.com/tomeshnet/ipfs-live-streaming/issues/79